src/main/java/org/keycloak/plugins/groups/services/GroupsService.java

```diff
@@ -34,6 +34,13 @@ public UserGroups userGroups() {
         return service;
     }
 
+    @Path("/vo-admin")
+    public VoAdminService voAdminService() {
+        VoAdminService service = new VoAdminService(session, realm);
```

Comment:
we could add here the check if the user calling this service is an admin. Otherwise, the call should be rejected.